README.md

```diff
@@ -88,7 +88,12 @@ const sdk = new ServerSDK({
 });
 
 const wallet = await sdk.createWallet("User Wallet");
-const signature = await sdk.signMessage(wallet.walletId, "Hello from Phantom!", NetworkId.SOLANA_MAINNET);
+const message = Buffer.from("Hello from Phantom!").toString("base64url"); // base64url encode
```

Comment:
would be nice if we can do this within the SDK, based on the chain. often very confusing for the user